kong/error_handlers.lua

```diff
@@ -59,6 +59,10 @@ return function(ctx)
   local status = kong.response.get_status()
   local message = get_body(status)
 
```

Comment:
It would be better if we add some comments in the code to explain why we do this.